.travis.yml

```diff
@@ -1,24 +1,18 @@
-sudo: required
-dist: trusty
-# Force travis to use its minimal image with default Python settings
+sudo: required 
+dist: trusty 
 language: generic
-compiler:
-  - gcc
-
+notifications:
```

Comment:
I am not aware of the intricacies for now, but if needed then we can roll back. @ipa-fmw